pkg/plugins/golang/deploy-image/v1alpha1/scaffolds/internal/templates/controllers/controller.go

```diff
@@ -72,21 +72,28 @@ import (
 
 	"context"
 	"time"
+	"fmt"
+	"github.com/go-logr/logr"
```

Comment:
```suggestion
```

We should use the log obtained from the context (provided by the controller-runtime only)